src/packageManager/fileDownloader.ts

```diff
@@ -66,6 +66,7 @@ async function downloadFile(
         agent: getProxyAgent(url, proxy, strictSSL),
         port: url.port,
         rejectUnauthorized: strictSSL,
+        timeout: 5 * 60 * 1000, // timeout is in milliseconds
```

Comment:
I chose a long timeout, because some of us have bad internet connections, I just don't want us to wait literally forever.

Reply:
I'm not sure if even this is long enough.  There might be some extremely slow connections out there and while before they could wait, now they'll never get to a working state.  Should we make this timeout configurable?

Reply:
I'm almost tempted to turn the download component into a VSCode progress window.  Then the user could cancel it if it was taking too long and not making any progress.

Reply:
That sounds like a good idea too. Will have a poke around